src/Compilers/Extension/project.json

```diff
@@ -1,8 +1,12 @@
 ﻿{
   "dependencies": {
     "EnvDTE": "8.0.0",
-    "Microsoft.VisualStudio.Shell.14.0": "14.3.25407",
-    "Microsoft.VisualStudio.Shell.Interop.10.0": "10.0.30319"
+    "Microsoft.VisualStudio.Shell.15.0": "15.0.26014-alpha",
+    "Microsoft.VisualStudio.Shell.Interop.8.0": "8.0.50727",
+    "Microsoft.VisualStudio.Shell.Interop.9.0":  "9.0.30729",
+    "Microsoft.VisualStudio.Shell.Interop.10.0": "10.0.30319",
+    "Microsoft.VisualStudio.Shell.Framework": "15.0.26014-alpha",
```

Comment:
Same goes here (https://www.nuget.org/packages/Microsoft.VisualStudio.Shell.Framework/15.0.26109-RC3) and the rest of our `-alpha` entries below.